app/abilities/report.js

```diff
@@ -4,11 +4,15 @@ import { Ability } from "ember-can";
 export default Ability.extend({
   canEdit: computed(
     "user.{id,isSuperuser}",
-    "model.{user.id,user.supervisors,task.project.reviewers,verifiedBy}",
+    "model.{user.id,user.supervisors,task.project.reviewers,verifiedBy,billed}",
     function() {
       return (
         this.get("user.isSuperuser") ||
-        (!(this.get("model.verifiedBy") && this.get("model.verifiedBy.id")) &&
+        (!(
+          this.get("model.verifiedBy") &&
```

Comment:
This line is not really necessary. At least as long as you are using `.get`.

Reply:
Fixed the tests and removed this